examples/todos/src/components/Todo.spec.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { shallow } from 'enzyme';
+import Todo from './Todo'
+
+const setup = (mockOnClick) => {
+  const wrapper = shallow(<Todo onClick={mockOnClick} completed={true} text="todo 1"/>);
+  return { wrapper };
+};
+
+describe('Todo component', () => {
+  const mockOnClick = jest.fn();
+
+  it('Should render Todo component', () => {
+    const { wrapper } = setup(mockOnClick);
+    expect(wrapper).toMatchSnapshot();
+  })
+
+  it('Should call click event', () => {
+    const { wrapper } = setup(mockOnClick);
+    const todoComponenet = wrapper;
```

Comment:
It so weird when you try to rename this variable, why don't we assign this name first time:
```
const { todoComponenet } = setup(mockOnClick);
```